### DISCUSSION TITLE
How to scroll in a "hover-dialog"? (LSP)

### DISCUSSION THREAD

**Discussion by Rudxain** [helix-editor/helix#9426] [Q&A] [answered]
While in `Insert` mode, the "on-hover" tooltips/docs (provided by the LSP) are usually cut-off, as they are longer than my terminal size (I use "big" fonts). Here's what I mean:
![image](https://github.com/helix-editor/helix/assets/76864299/0b430e92-189a-48ab-a162-2c238fa234cb)

I am forced to reduce the font-scale, in order to read the docs or simply get an MDN link. How can I focus into that dialog to navigate it with j/k (or arrows)?

**Comment by TornaxO7** [comment]
AFAIK it's not implemented yet

**Comment by kirawi** [comment]
`C-d` and `C-u`

**Reply by Rudxain** [reply]
WDYM? I tried pressing `ctrl+c+u`, didn't work. Tried `alt+c+u`, it didn't work either

**Reply by kirawi** [reply]
`C-d` is `ctrl-d` and `C-u` is `ctrl-u`

**Reply by Rudxain** [reply]
I tried that, it didn't work :(

**Reply by Rudxain** [reply]
Wait, it works in normal mode (not insert)

**Comment by the-mikedavis** [comment, marked as answer]
Scrolling that completion documentation isn't supported yet, see https://github.com/helix-editor/helix/issues/3062

Hover (`<space>k` over a symbol) is scrollable though with `C-u` and `C-d` as @kirawi says

**Reply by Rudxain** [reply]
Thanks for the info. I'm sorry for posting a dupe. I tried searching, but couldn't find similar issues

**Comment by leandrobbraga** [comment]
Can we configure this hover window size to make it larger?

**Reply by Rudxain** [reply]
IDK, but #15187 can help
